src/vis-preview.py

```diff
@@ -42,39 +43,84 @@ def main():  # pragma: no cover
     parser.add_argument(
         '--to_json', action='store_true',
         help='Output viewconf, rather than open in browser.')
+    # parser.add_argument(
+    #     '--parent_uuid', action='store_true',
+    #     help='Parent uuid for the dataset',
+    #     default=None)
+    parser.add_argument(
+        '--epic_uuid', metavar='UUID',
+        help='uuid of the EPIC dataset',
+        default=None)
+
+    #
+    # parser.add_argument(
+    #     '--epic_builder',  action='store_true',
+    #     help='Whether to use the epic_builder or not',
+    #     default=None)
 
     args = parser.parse_args()
     marker = args.marker
+    # epic_builder = args.epic_builder
+    epic_uuid = args.epic_uuid
+    # parent_uuid = args.parent_uuid # this may not be needed, as the --url provides the parent dataset json?
```

Comment:
The `parent_uuid` is necessary for visualizing image pyramid support datasets, since the logic in `builder_factory` expects `parent is not None` for those cases and performs an assaytype lookup on the parent to determine which specific image pyramid builder to use (to handle evolution of how image pyramids were handled over time): https://github.com/hubmapconsortium/portal-visualization/blob/main/src/portal_visualization/builder_factory.py#L75-L76